### ISSUE TITLE
[Feature]: The name simplex is confusing and ambiguous. 

### ISSUE THREAD

**Issue by rashomonOS** [simplex-chat/simplex-chat#3676]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### App version

All

### Feature

IMHO the app name needs to be changed. Sad as it is, adoption is made easier by easily recognisable and pronounced branding, and the current name is ambiguous and confusing.

* There is general confusion about how to pronounce it - sim-plecks? Sim-pull-ecks? The branding implies the latter but the package name (and the fact not many people capitalise branding in texting any more) implies the former. 
 
Whichever it is, though, only leads to further confusion over what the app does, e.g.:

* Simple-X (sim-pull-ecks) has been inferred by a few I've evangelised to as a reference to a famous micro-blogging site formerly represented by a small blue bird, and so might be better suited to an app that makes it easier to use a federated equivalent like Mastodon rather than a chat app similar to Signal. Most assumed it was something like Tweetdeck.

* Simplex (sim-plecks) is an existing term in signalling, meaning that sender can send but not receive data, so in this term this name would better suit a broadcast protocol that can't be replied to, which is not the case with SimpleX. 

* The word simplex is probably better known to the general public via the herpes simplex virus, which affects about 60% of adults so this will be the most frequent association the general public make. This is awkward and a source of adolescent humour at best, but at worst will probably lead a minority of potential users to assume that simplex is a virus because that's the existing association for most people. 

All of these issues have popped up when I've evangelised the app in the last month. And yes, everybody has noticed the herpes association.

My suggestion: a different name more evocative of the purpose of the app that it is easier to pronounce at first glance.

**Comment by epoberezkin** [comment]
Thanks for such a detailed analysis and for helping people adopt it :)

SimpleX name has been around long before the blue bird was killed 😢, and we have got used to calling it Simple-X, following our users, and the branding was supposed to mean both simple to use and private & secure at the same time - the combination we are still not finding in one product.

And yes, originally it was simplex of course, due to unidirectional network design - each relay provides unidirectional queues, and each conversation usually happens via different relays, each conveying messages only in one direction (client will always choose another relay for reply queue when it is available in configuration).

We're of course aware of simplex family of viruses (it's not only herpes, and it is dormant in about 90-95% of the population these days I think), and of the associated humour based on the popular culture, and confusion that it's associated with STDs - it's been certainly more than a decade since it's no longer the case, as most of us has this virus.

Our current focus is building the most private and secure messenger we can build, that is convenient to use at the same time, becoming a category leader for privacy enthusiasts and users who need (rather than want) privacy. These users like the brand, and it's highly recognisable to many of them, so we do think that re-branding right now will do more harm than good. Also, the product is still early stage, and there are many rough edges, so mass-market users attracted by a good brand are likely to be disappointed in some other way - it's still full of compromises.

But once the goal of becoming a category leader is achieved, and technology is stable (2025? 6?) we are indeed considering re-branding to help adoption of mass-market product.

Do you have any specific brand suggestions btw?

Feel free to connect directly if you have some good ideas, no need to share publicly :)

**Comment by maxtar** [comment]
I like this name. :)

**Comment by Rubber-Duckie** [comment]
Its better than Herpes... 
You could call it HSV for short ?

**Comment by Rubber-Duckie** [comment]
Its better than Herpes... 
You could call it HSV for short ?
Frankly, this name is detrimental to your search engine visibility. The chat app will get lost among the myriad of simplex-related results. Simplex is not only a common word, but also a medical term, among other things, so a rename might be advisable.

Simplex, as a communication concept, implies a very restricted form of interaction, which seems ironic given the intricate challenges of creating a truly privacy-oriented messenger in an increasingly hostile environment.

'Multiplex' would be a more suitable name if they are adamant about the ontology

**Comment by xxxcrow** [comment]
I never call it simplex with anyone, we all just say "the magic messenger" or just "the messenger"

**Comment by Rubber-Duckie** [comment]
Agreed, best avoided. If you asked them if they had simplex, they might say its non of your business. Awkward.
Admitting you had it yourself, they could ask A or B ? Again Awkward.

**Comment by pedrodelbosque** [comment]
I either connect with users though a link I send them, or tell them to scan my QR. Both options will open the SimpleX website if the app is not installed, which the user will promptly do. This is why I've never had to struggle with spelling the apps name. As this step is mandatory, I don't see any step in which the spelling of the apps name would matter, marketing aside.

Side note, as non-native english speaker, I had never heard of the virus family before, neither have the few people I asked. Seems like everyone knows it as just 'herpes'.

**Comment by kimg45** [comment]
Eh can always just pronounce it Simple X if you want. No one I've evangelized to have made that connection lol. I kinda like the name anyway.

**Comment by Sepero** [comment]
I also pronounce it "Simple-Eks".

Though I agree that the SEO can be a genuine issue.

**Comment by Sepero** [comment]
Unique names that may have good/unique SEO:
GuardIQ
Guardra
Hushox
Hushon
Maskix
ShadeX

**Comment by eyeinsi** [comment]
simp-lex

**Comment by summoner001** [comment]
CrypteX :)
